hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -811,6 +896,30 @@ func reconcileUserDataSecret(userDataSecret *corev1.Secret, nodePool *hyperv1.No
 	return nil
 }
 
+// jmencak
+func reconcileTunedConfigMap(tunedConfigMap *corev1.ConfigMap, nodePool *hyperv1.NodePool, config string) error {
```

Comment:
This seems to be copied from reconcileTokenSecret which is a unique secret source of truth for Node release+config and watched by ignition server token secret controller. This new secret won't have any impact on the existing workflow.

You can see more details about the workflow here https://github.com/openshift/enhancements/blob/master/enhancements/hypershift/node-lifecycle.md

Reply:
Resolving this as the hash / version logic has been removed for these Tuned configmaps.